src/pages/LandingPage.js

```diff
@@ -71,6 +71,7 @@ const LandingPage = () => {
     ]),
     isTerra() && div({
       style: {
+        backgroundColor: '#191a1c',
```

Comment:
Can we make this use the color variables? (No worries if not, just wanted to check)

Reply:
The background is currently that virus image with a dark background on a gradient from left to right. I was trying to match the color of the left-most area so that the progressive enhancement would look reasonable for users who see the color before the image.  As far as I know there isn't a color name defined for that background color since this seems to be the only place using it.

Same thing with the other hexagon background image: I was trying to match the color of the image exactly so you don't see a line at the transition.

Reply:
Fair enough! I am comfortable with that!

Reply:
Tell you what: I'll add a comment explaining where that "magic" color code comes from. :)